spec/models/equipment_model_spec.rb

```diff
@@ -286,41 +339,62 @@
 
     context 'methods involving reservations' do
       # @model and @category are already set.
+      ACTIVE = [:valid_reservation, :checked_out_reservation]
```

Comment:
Oh whoops, I wrote this before I got that far and didn't connect the dots, thanks!